it/src/main/java/com/google/cloud/teleport/it/bigquery/BigQueryVerifiers.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.it.bigquery;
+
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.BigQueryOptions;
+import com.google.cloud.bigquery.QueryJobConfiguration;
+import com.google.cloud.bigquery.TableResult;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+/** A class that implements methods to interact with BigQuery and verify test behavior. */
+public class BigQueryVerifiers {
```

Comment:
Could we tie this to the resource manager, so that we can maybe do things like make sure only test tables are queried? Having some class we can mock and inject into this could also make it possible to test.

Reply:
done